javascript/node/code-style-recommendations/styling-brackets.md

```diff
@@ -19,14 +15,20 @@ tags:
 aspects:
   - introduction
 
+links:
+  - '[MDN - ASI](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Lexical_grammar#Automatic_semicolon_insertion
+){documentation}'
+  - '[MDN - Block](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Statements/block){documentation}
 
 ---
 # Styling Brackets
 
 ---
 ## Content
 
-Styling is something that shouldn't influence the performance of the code in any way, and differs from team to team. It's rare that developers agree on camel case syntax versus underscore naming conventions, tabs versus spaces, or brackets position. However, for JavaScript in general and Node in particular, brackets location isn't just a debate. Let's look at  an example:
+The style of the code shouldn't influence its performance in any way. The guidelines for what is the "right" style differ from team to team, and are primarily subjective. It's common that developers would disagree on using `camelCase` versus `kebab_case` naming conventions, tabs versus spaces, or various positions of braces or brackets. 
```

Comment:
The comma after "team to team" is unnecessary.
Suggestion:
It's common that developers... -> Commonly, developers...

Reply:
```suggestion
The style of the code shouldn't influence its performance in any way. The guidelines for what is the "right" style differ from team to team and are primarily subjective. Commonly, developers would disagree on using `camelCase` versus `kebab_case` naming conventions, tabs versus spaces, or various positions of braces or brackets. 
```